### ISSUE TITLE
FBLANK is a blanking period

### ISSUE THREAD

**Issue by jeffythedragonslayer** [undisbeliever/snesdev-notes#10]
This sentence threw me for a loop:

> When the force-blank bit is set, the PPU will disable the screen output and allow the CPU to safely access the PPU registers outside the blanking periods.

I think it should probably end in "outside the vertical blanking period" because FBLANK is also a blanking period.

**Comment by undisbeliever** [comment]
I do not consider force-blank to be a blanking period, it does not have a specific start or end time nor it is a part of the TV sync pulse.

I am using the phrase 'blanking periods' to refer to both the horizontal blanking and vertical blanking periods.

I am undecided as to whether I should change the sentence or not.  I'm leaning towards not, as I have seen the phrase 'blanking periods' refer to only the horizontal and vertical blanking periods in the past.

I may add a link to a terminology/glossary page instead.  Would that suffice?

**Comment by jeffythedragonslayer** [comment]
Yes; the sentence just confused me because some of the first snes documentation I read lumped `VBLANK`, `HBLANK`, and `FBLANK` together
